### ISSUE TITLE
Bad lines sent when statsd stats are enabled

### ISSUE THREAD

**Issue by mivok** [chef/chef-server#679]
If `estatsd['protocol'] = 'statsd'` is set in chef-server.rb, and statsd are sent to an etsy statsd server, the following is logged for every packet sent by chef to statsd:

```
29 Dec 22:59:34 - DEBUG: Bad line: 1 in msg "1|158"
```

Where the second number in `1|158` changes (but seems the be between 150 and 700) and the first number is always 1.

Doing a tcpdump of the statsd packets shows this line as the first line in the statsd packet. All other lines look valid. The following is a sample packet sent by erchef:

```
1|690
chefAPI.application.byStatusCode.200:1|c
chefAPI.example-host.byStatusCode.200:1|c
chefAPI.application.allRequests:13|ms
chefAPI.example-host.allRequests:13|ms
chefAPI.application.byRequestType.data.GET:13|ms
chefAPI.upstreamRequests.rdbms:4|ms
chefAPI.upstreamRequests.authz:5|ms
chefAPI.upstreamRequests.rdbms.chef_sql.fetch_actors_by_name:1|ms
chefAPI.upstreamRequests.rdbms.chef_sql.fetch:2|ms
chefAPI.upstreamRequests.rdbms.chef_sql.fetch_org_metadata:1|ms
chefAPI.upstreamRequests.authz.oc_chef_authz.is_authorized_on_resource:5|ms
chefAPI.application.byRequestType.data.GET.upstreamRequests.rdbms:4|ms
chefAPI.application.byRequestType.data.GET.upstreamRequests.authz:5|ms
```

This shows up for bifrost and reporting stats too.

**Comment by stevendanna** [comment]
The `1|690` is part of the estatsd format.  We need to thread the protocol version through to here:

https://github.com/chef/stats_hero/blob/master/src/stats_hero_sender.erl#L58-L61

**Comment by stevendanna** [comment]
Upstream fix: https://github.com/chef/stats_hero/pull/13

**Comment by stevendanna** [comment]
This has been fixed since 12.5.0 (commit 17320b58).  Thanks for reporting it and sorry for any trouble it caused!
